lib/eval_on_dart_library.dart

```diff
@@ -63,4 +96,103 @@ class EvalOnDartLibrary {
         print('Unrecognized error: $e');
     }
   }
+
+  Future<Library> getLibrary(LibraryRef instance, ObjectGroup isAlive) {
+    return getObjHelper(instance, isAlive);
+  }
+
+  Future<Class> getClass(ClassRef instance, ObjectGroup isAlive) {
+    return getObjHelper(instance, isAlive);
+  }
+
+  Future<Func> getFunc(FuncRef instance, ObjectGroup isAlive) {
+    return getObjHelper(instance, isAlive);
+  }
+
+  Future<Instance> getInstance(
+    FutureOr<InstanceRef> instanceRefFuture,
+    ObjectGroup isAlive,
+  ) async {
+    return await getObjHelper(await instanceRefFuture, isAlive);
+  }
+
+  /// Public so that other related classes such as InspectorService can ensure their
+  /// requests are in a consistent order with requests which eliminates otherwise
+  /// surprising timing bugs such as if a request to dispose an
+  /// InspectorService.ObjectGroup was issued after a request to read properties
+  /// from an object in a group but the request to dispose the object group
+  /// occurred first.
+  /// <p>
+  /// The design is we have at most 1 pending request at a time. This sacrifices
+  /// some throughput with the advantage of predictable semantics and the benefit
+  /// that we are able to skip large numbers of requests if they happen to be
+  /// from groups of objects that should no longer be kept alive.
```

Comment:
Another re-phrasing nit: 

/// With this design, we have at most 1 pending request at a time. This sacrifices
/// some throughput, but we gain the advantage of predictable semantics and the
/// ability to skip large numbers of requests from object groups that should no longer
/// be kept alive.

Reply:
thanks. rephrased.